installer.tpl.sh

```diff
@@ -235,8 +235,15 @@ install_apt ()
   curl -L "${gpg_key_url_modules}" | apt-key add -
 
   rm -f /etc/apt/sources.list.d/*tarantool*.list
-  echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/${repo_path}${ver}${gc64}/${os}/ ${dist} main" > ${apt_source_path}
-  echo "deb-src https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/${repo_path}${ver}${gc64}/${os}/ ${dist} main" >> ${apt_source_path}
+  # For 3 series we use static builds and we can use one repository
+  # for all deb-based and rpm-based systems
+  if [ "${ver}" = 3 ]; then
+    echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/${repo_path}${ver}${gc64}/linux-deb/ static main" > ${apt_source_path}
+    echo "deb-src https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/${repo_path}${ver}${gc64}/linux-deb/ static main" >> ${apt_source_path}
+  else
+    echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/${repo_path}${ver}${gc64}/${os}/ ${dist} main" > ${apt_source_path}
```

Comment:
It is a copy-paste. It should be done in a more elegant way.

Reply:
done